core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -61,4 +64,22 @@ private Cards() {}
     static Set<Card> getStandardDeck() {
         return STANDARD_DECK;
     }
+
+    static Stack<Card> getShuffledStandardDeck(){
+        return getStandardDeck().stream()
+                .map(c -> new CardPair(RANDOM.nextInt(10_000_000), c))
+                .sorted((o1, o2) -> Integer.compare(o1.position, o2.position))
+                .map(cp -> cp.card)
+                .collect(Collectors.toCollection(Stack::new));
+    }
+
+    private static class CardPair{
+        public final int position;
+        public final Card card;
+
+        private CardPair(int position, Card card) {
```

Comment:
Nit: consider the sequence of the parameters for readability purposes.

In my brains in particular - _"there is a card at position"_ as opposed to _"there is a position at which is a card"_